SI/Sem.02/Pract.02/Task07.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+int main() {
+	char input;
+
+	std::cin >> input;
+
+	if (input >= 'A' && input <= 'Z') {
+		std::cout << char(int(input) + 32) << std::endl;
```

Comment:
the casts to int and char are unneeded

Reply:
If I remove both casts I end up with this code "std::cout << input + 32 << std::endl;" and if the input is "A" the output is 97, not the character a. So then maybe I do need at least the char cast?

Reply:
Ping us tomorrow to discuss it.